src/sacn/receiver.c

```diff
@@ -480,6 +546,18 @@ SacnReceiver* create_new_receiver(const SacnReceiverConfig* config)
 
   receiver->socket = ETCPAL_SOCKET_INVALID;
 
+  if (config->netints != NULL)
+  {
+    receiver->netints = calloc(config->num_netints, sizeof(SacnMcastNetintId));
```

Comment:
I understand the netints now need to be saved, so that change_universe will work. This calloc() is not guarded by an #ifdef SACN_DYNAMIC_MEM right now. You will need to tweak the SacnReceiver struct to have something like:

```c
  /* (optional) array of network interfaces on which to listen to the specified universe. If NULL,
   * all available network interfaces will be used. */
#ifdef SACN_DYNAMIC_MEM
  const SacnMcastNetintId* netints;
#else
  const SacnMcastNetintId netints[SACN_MAX_NETINTS];
#endif
  /* Number of elements in the netints array. */
  size_t num_netints;
```

Then change this code so that it either does the calloc if SACN_DYNAMIC_MEM==1, or simply fills in the static array if SACN_DYNAMIC_MEM==0. It should also fail if the calloc fails, or if too many netints are specified for the static array. This logic is probably worth refactoring into its own small static function to avoid polluting this function with #ifdefs.

Reply:
Will work on this next week.

Reply:
The code for this has just been committed.